tests/acceptance/test_performance_trace_detail.py

```diff
@@ -72,9 +75,9 @@ def setUp(self):
         self.login_as(self.user)
 
         self.day_ago = before_now(days=1).replace(hour=10, minute=0, second=0, microsecond=0)
-        self.trace_id = uuid4().hex
+        self.trace_id = "fd3fc65b2bb54c349f50337143234a83"  # make this trace id static
```

Comment:
nit: (and it really is a nit) I prefer how we do "c" * 32 cause its far more obvious its not random

Reply:
That's a good point, will do that to make the intent obvious.